downstream/modules/platform/proc-controller-ldap-security.adoc

```diff
@@ -0,0 +1,10 @@
+[id="proc_controller-ldap-security_{context}"]
+
+= controller-ldap-security
+
+
```

Comment:
I would suggest adding a brief, concise abstract to this to help the user understand the "why" for this new topic. Might want to verify with the SME.